### PR TITLE
Google Recaptcha implementation

### PR DESCRIPTION
**Pull request by aleksandargj**
Option to protect from spam with Google Recaptcha

### CODE REVIEW

File: public/js/cinode-recruitment-public.js [hunk 1]
```diff
@@ -34,29 +36,34 @@ jQuery(document).ready(function ($) {
         formData.append("linkedInUrl", linkedInUrl.val());
         formData.append("state", 0);
         formData.append("currencyId", currencyId);
-        if (pipelineId){
+
+        if (pipelineId) {
```
Comment (antur84) [inline public/js/cinode-recruitment-public.js:40]: `pipelineId` never defined anywhere, is that correct?

File: public/js/cinode-recruitment-public.js [hunk 2]
```diff
@@ -34,29 +36,34 @@ jQuery(document).ready(function ($) {
...
           formData.append("pipelineId", pipelineId);
+        } else if (multiplePipeline) {
+          formData.append("pipelineId", multiplePipeline);
         }
-        
-        if (pipelineStageId){
-        formData.append("pipelineStageId", pipelineStageId);
+
+        if (pipelineStageId) {
```
Comment (antur84) [inline public/js/cinode-recruitment-public.js:46]: `pipelineStageId` never defined anywhere, is that correct?

File: public/js/cinode-recruitment-public.js [hunk 3]
```diff
@@ -34,29 +36,34 @@ jQuery(document).ready(function ($) {
...
+          formData.append("pipelineStageId", pipelineStageId);
+        } else if (multipleStageId) {
+          formData.append("pipelineStageId", multipleStageId);
         }
-        if (recruitmentManagerId){
+
+        if (recruitmentManagerId) {
```
Comment (antur84) [inline public/js/cinode-recruitment-public.js:52]: Guess I don't know how this works, where do all these come from? They just look undefined to me :)

File: README.md [hunk 4]
```diff
@@ -20,6 +20,15 @@ This section describes how to install the plugin and get it working.
 
 ## Releases
 
+### 1.4.0
+https://github.com/Cinode-Labs/cinode-recruitment/releases/tag/v1.4
+
+Update with the following feature:
+* Option to protect from spam with Google Recaptcha
+* Optional field set Available from date
+* Optional field for select Pipeline, Multiple pipelines
```
Comment (antur84) [inline README.md:29]: What is "multiple pipelines" feature? Is it the ability to put candidates i certain pipe/stage depending on conditionals? Maybe slighly expand text, unless widely known what multiple pipelines means.

**Review by antur84 — COMMENTED** [review]
looked at the JS files and docs, had minor questions.

**Comment by antur84** [comment]
Please resolve the conflicts as well 👍

File: public/js/cinode-recruitment-public.js (hunk 1 above)
Comment (aleksandargj) [reply public/js/cinode-recruitment-public.js:40]: pipelineId is defined inside form, and its fetched from backend

File: public/js/cinode-recruitment-public.js (hunk 2 above)
Comment (aleksandargj) [reply public/js/cinode-recruitment-public.js:46]: pipelineStageId is defined inside form, and its fetched from backend

File: public/js/cinode-recruitment-public.js (hunk 3 above)
Comment (aleksandargj) [reply public/js/cinode-recruitment-public.js:52]: all of them are defined inside form, and its fetched from backend

File: README.md (hunk 4 above)
Comment (aleksandargj) [reply README.md:29]: Candidate can select option from dropdown list of available Pipelines

**Comment by aleksandargj** [comment]
Because of conflicts, I will create new PRs.

File: cinode-recruitment.php [hunk 5]
```diff
@@ -516,6 +471,8 @@ function cinode_recruitment_shortcode($atts = [])
 		'recruitmentsourceid' => 0,
 		'campaigncode' => 0,
 		'currencyid' => 1,
+		'multiplepipelines' =>'',
+		'multiplepipelinestageid' => 0,
```
Comment (antur84) [inline cinode-recruitment.php:475]: Could `multiplepipelinestageid` have been named `multiplepipelines_stageid`?

**Comment by aleksandargj** [comment]
Yes, you can close this PR

**Pull request closed without merging**